portfolio/src/main/webapp/blogpage.css

```diff
@@ -0,0 +1,100 @@
+body {
+  margin: auto;
+  font-family: 'Montserrat', serif;
+}
+
+.landing-section {
+  background-color: cyan;
+  height: 100vh;
+  width: 100vw;
```

Comment:
same here - typically, we shouldn't need to set the width. try setting the display properties instead!